apps/web-staking/src/services/web3.service.ts

```diff
@@ -712,8 +712,11 @@ export const updateRequestClaimed = (network: NetworkKey, walletAddress: `0x${st
 		throw new Error('Invalid requestIndex');
 	}
 
-	cachedUnstakes[requestIndex].open = false;
-	cachedUnstakes[requestIndex].completeTime = Date.now();
+	if (cachedUnstakes[requestIndex]) {
+		cachedUnstakes[requestIndex].open = false;
+		cachedUnstakes[requestIndex].completeTime = Date.now();
+	}
+	
```

Comment:
This seems strange to me, how come the `cachedUnstakes[requestIndex]` would be undefined, if it gets diplayed in the UI as claimable that has to be from the localStorage.